### ISSUE TITLE
Add new chat messages logging

### ISSUE THREAD

**Issue by Freewalkr** [slskd/slskd#976]
It would be great to have some way of learning about new messages without checking web interface.
I think the most simple way for developers to do it is to log something like "New message received from <user>".
Then log analyzers can be used to send notifications (out of slskd scope == simple and versatile).
For example (in my setup), Loki will grab logs, find "new message" line and use Alertmanager to send a message to my Telegram.

**Comment by vfedetz** [comment]
I agree, some way to be notified about new chats besides checking the UI would be useful.
